x/pool-incentives/keeper/keeper.go

```diff
@@ -113,7 +112,7 @@ func (k Keeper) CreateConcentratedLiquidityPoolGauge(ctx sdk.Context, poolId uin
 		// lockQueryType as byTime. Although we donot need this check, we still cannot pass empty struct.
 		lockuptypes.QueryCondition{
 			LockQueryType: lockuptypes.ByTime,
-			Denom:         appparams.BaseCoinUnit,
+			Denom:         sdk.DefaultBondDenom,
```

Comment:
note to reviewers: we technically donot care about the denom since, we donot use it while creating CL gauges. so i changed it to "stake" to make simulator testing easier. We cannot leave it empty as it is a require field in `CreateGauge`